### DISCUSSION TITLE
how do i select the recipes shown when not logged in

### DISCUSSION THREAD

**Discussion by sdetweil** [mealie-recipes/mealie#3387] [General]
my userid is the default group
my sisters are in their own groups

when they are logged out, they can see sll my recipes, not theirs.

**Comment by Kuchenpirat** [comment]
Please use the provided templates when creating an issue. 

Logged out, if the user does access the instance directly and not a subpage, you get redirected to the default group.
They would have to navigate to the subpage `/g/<their_group_slug>`

**Comment by sdetweil** [comment]
none of the templates applied  do i used the link to empty

**Comment by Kuchenpirat** [comment]
i'll convert this into a discussion since it does not seem like a issue. 

If the user is not logged in we would have no way of matching them with their group. So we just redirect them to the default group if it is public. Your sisters would either have to log-in or to navigate to their group slug directly. 

To extend time between log-ins you can set the `TOKEN_TIME` env var to a number as large as you like, which will set the number the token is valid for in hours. 

https://docs.mealie.io/documentation/getting-started/installation/backend-config/
